### PR TITLE
New License: FSFULLRWD

### LINKED ISSUE #1556: New license request: FSFULLRWD [SPDX-Online-Tools]
**1.** License Name: FSF Unlimited License (With License Retention and Warranty Disclaimer)
**2.** Short identifier: FSFULLRWD
**3.** License Author or steward: Free Software Foundation
**4.** Comments: It is a minimalist highly permissive license that is evidently inserted on Makefile.in files generated by GNU Automake. It resembles FSFULLR except it adds a typical GNU-style warranty disclaimer. I found it in one Fedora source package (cockpit).
**5.** Standard License Header: 
**6.** License Request Url: http://tools.spdx.org/app/license_requests/152
**7.** URL(s): https://lists.gnu.org/archive/html/autoconf/2012-04/msg00061.html (first reference)
**8.** OSI Status: Not Submitted
**9.** Example Projects: https://kojipkgs.fedoraproject.org//packages/cockpit/270/1.fc36/src/cockpit-270-1.fc36.src.rpm (extract the source archive)

### PR DESCRIPTION
**Pull request by jlovejoy**
fixes #1556 

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

File: src/FSFULLRWD.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="FSFULLRWD"
+   name="FSF Unlimited License (With License Retention
+   and Warranty Disclaimer)" listVersionAdded="3.19">
+      <crossRefs>
+         <crossRef>https://lists.gnu.org/archive/html/autoconf/2012-04/msg00061.html (first reference)</crossRef>
```
Comment (swinslow) [inline src/FSFULLRWD.xml:7]: Do we require the `<crossRef>` to be a URL? I'm assuming this is what the checker uses to see if the link is live... If so, then we might want to remove the `(first reference)` comment at the end of this.

**Review by swinslow — COMMENTED** [review]
Hi @jlovejoy, just one question on this one (in the crossRef section). Thanks!

File: src/FSFULLRWD.xml (hunk 1 above)
Comment (jlovejoy) [reply src/FSFULLRWD.xml:7]: good point, not sure, but will remove!

**Review by swinslow — COMMENTED** [review]
Looks like this is failing because 1a36f16 inadvertently removed the `</crossRef>` from the end of the cross reference... I'll edit and reinsert.

**Review by swinslow — APPROVED** [review]
Now passing the tests, so I'm merging this.

**Pull request merged**